website/blog/2018-05-23-1.13.0.md

```diff
@@ -0,0 +1,948 @@
+
+<!--truncate-->
+
+## Highlights
+
+### API/CLI
+
+#### Prettier works in the browser!
+
+This has been long wanted by the team and our users and we're finally announcing Prettier 1.13 has first-class support for running in the browser. In the past it required several hacks, but as of today you can just load the core bundle and the parsers you want to use.
+
+<!-- prettier-ignore -->
+```html
+<script src="https://unpkg.me/prettier@1.13.0/standalone.js"></script>
+<script src="https://unpkg.me/prettier@1.13.0/parser-babylon.js"></script>
+<script src="https://unpkg.me/prettier@1.13.0/parser-graphql.js"></script>
+<script type="text/javascript">
+  $("#format").onClick(function() {
+    var input = $("#input").val();
+    $("#input").val(prettier.format(input, { parser: "babylon", plugins: prettierPlugins }));
+  });
+</script>
+```
+
+Read more about it in the [docs](browser.md).
+
+#### Don't default to the JavaScript parser ([#4528] by [@duailibe])
+
+In Prettier 1.12 and earlier, when running Prettier on a file without a file extension, Prettier would assume the file contained JavaScript. This lead to a couple of confusing bugs, namely:
+
+* If you ran prettier on your `.git` directory, it would reformat files and break your git repo.
+* If you tried to run prettier on html files, it would sometimes throw an error, but other times parse the HTML as JSX, and format it as if it was JSX (which was often unsafe).
+
+In Prettier 1.13, we are changing this behavior. If Prettier cannot determine what language a file is from its extension, and the user has not manually specified a parser to use, Prettier will skip the file.
+
+If you need to format files with unsupported or nonstandard extensions, you can use the `--parser` option to specify which parser to use. If you are piping file contents into `stdin`, make sure to include `--stdin-filepath` so that prettier can use the file extension to detect what language to parse the input as.
+
+When using the JavaScript API, `prettier.format` and `prettier.formatWithCursor` will now throw an error if you do not include either `parser` or `filepath` in the options you pass in. It will also throw an error if you include `filepath` but not `parser` and the correct parser cannot be inferred from the file path.
+
+#### `prettier.getFileInfo()` method and `--file-info` CLI option ([#4341] by [@kachkaev])
+
+When invoking Prettier from the API, there was no way to find out if a file should be ignored or not (for `.prettierignore`) so editor integration plugins would have to implement this logic. Also, there was no way to find out if a file was supported by Prettier, so users would need to call Prettier anyway. In 1.13 there's a way to check if a parser will be inferred (and which one) and if the file is ignored or not before actually calling `format`.
+
+You can read more about this in the [docs](api.md#prettiergetfileinfofilepath-options).
```

Comment:
This link is broken :(

Reply:
Looks like docusaurus won't change these links in blog posts